index.html

```diff
@@ -9,33 +9,43 @@
 <body>
     
     <header class="first">
-        <div><img src="./images/Welcome.png" alt="welcome button at the top left"></div>
-       <div> <img src="./images/Union.png" alt="a union of three vectors"> </div>
+
+    <div> 
+        <ul class="header-first">
+         <li>Portfolio</li>
+         <li>About </li> 
+         <li>Contact </li>
+        </ul>
+     </div>
+
+
     </header>
     <br>
     <section class="second">
-      <h1>  Hey there. I’m Mariane <span class="green" >I’m a software developer</span></h1>
-    <p>
+      <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer</span></h1>
+    <p class="second-text">
```

Comment:
- [ ] Kindly maintain the design of the mobile version for the mobile version

## Your Design
![image](https://user-images.githubusercontent.com/86200356/193415886-8a37a16b-f6dc-423c-a9d2-21fe6c3760ab.png)

## Figma
![image](https://user-images.githubusercontent.com/86200356/193415985-49cdd269-6773-48a9-8ba1-a11898f6df02.png)